code/src/reducers/reusable.js

```diff
@@ -0,0 +1,48 @@
+import { game } from "./game";
+import { ui } from "./ui";
+
+export const getStartGame = () => {
+  const startGameURL = "https://wk16-backend.herokuapp.com/start";
```

Comment:
Nice that you created a variable for the URL, this could also be imported from a separate URL file.